cmd/manager/main.go

```diff
@@ -76,6 +68,22 @@ func main() {
 
 	pflag.Parse()
 
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
```

Comment:
Can't we just call `version.Print()` here? I don't see a reason do something different here.

Reply:
Agreed, either make version to include the golang version info or leave it as is and print the return value of `version.GetVersion()`

Reply:
can use `version.Print()` here.

Reply:
as decided in ParkingLot. Decided to keep it as it is